queries/cpp/injections.scm

```diff
@@ -1 +1,2 @@
+; inherits: c
```

Comment:
nope, we can't inject C in C++ proproc macros!

Reply:
ah, good catch!